### DISCUSSION TITLE
Support for Edge Cache API

### DISCUSSION THREAD

**Discussion by tonprince** [vercel/community#108] [Help] [answered]
Hi
In several discussions and bug reports there is a need to have a way to invalidate the edge cache for serverless functions and resources on the CDN. I would love to see a cache API (similar to Cloudflare's cache API) where we can laser-precise invalidate URLs without having to redeploy the whole application. In my humble opinion that is the last missing piece (together with the recently published edge functions) to build a top of the edge web application with dynamic content and flexible caching.

Many thanks for your great work :-)

**Comment by ErnestBrandi** [comment]
I agree, not having a way to invalidate cache on demand (on content update for instance) is a possible pain in user experience (not having fresh data).

**Comment by leerob** [comment]
This is something we're exploring 👍

**Comment by leerob** [comment, marked as answer]
Let's continue the conversation here! https://github.com/vercel/community/discussions/268
